package.json

```diff
@@ -76,7 +76,7 @@
     "@types/tinycolor2": "^1.4.3",
     "@vitejs/plugin-react": "^4.1.0",
     "ag-grid-community": "^32.0.0",
-    "ag-grid-enterprise": "^32.0.0",
+    "ag-grid-enterprise": "^32.2.0",
     "ag-grid-react": "^32.0.0",
```

Comment:
I think all the packages should be the same version, maybe we should keep these as they are and add a resolution to 32.2.1